schemas/family_history.json

```diff
@@ -0,0 +1,154 @@
+        "primaryId": true,
+        "foreignKey": "sample_registration.program_id",
+        "displayName": "Program ID"
+      },
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "meta": {
+        "validationDependency": true,
+        "primaryId": true,
+        "foreignKey": "sample_registration.submitter_donor_id",
+        "displayName": "Submitter Donor ID"
+      },
+      "restrictions": {
+        "required": true,
+        "regex": "#/regex/submitter_id"
+      }
+    },
+    {
+      "description": "Unique identifier of the relative, assigned by the data provider.",
+      "name": "family_relative_id",
+      "valueType": "string",
+      "meta": {
+        "displayName": "Family Relative ID",
+        "notes": "This field is required to ensure that family members are identified in unique records.  Ids can be as simple as an incremented numeral to ensure uniqueness."
+
+      },
+      "restrictions": {
+        "required": true,
+        "regex": "#/regex/submitter_id"
+      }
+    },
+    {
+      "description": "Indicate if patient has any genetic relatives with a history of cancer. (Reference: NCIt C159104, caDSR CDE ID 6161023)",
+      "name": "relative_with_cancer_history",
+      "restrictions": {
+        "codeList": "#/list/yes_no"
+      },
+      "valueType": "string",
+      "meta": {
+        "displayName": "Relative with Cancer History"
+      }
+    },
+    {
+      "description": "Indicate genetic relationship of the relative to patient. (Reference: caDSR CDE ID 2179937)",
+      "name": "relationship_type",
+      "restrictions": {
+        "codeList": [
+          "Aunt",
+          "Brother",
+          "Cousin",
+          "Daughter",
+          "Father",
+          "Grandchild",
+          "Grandfather",
+          "Grandmother",
+          "Maternal Aunt",
+          "Maternal Cousin",
+          "Maternal Grandfather",
+          "Maternal Grandmother",
+          "Maternal Half-brother",
+          "Maternal Half-sister",
+          "Mother",
+          "Nephew",
+          "Niece",
+          "Other",
+          "Paternal Aunt",
+          "Paternal Cousin",
+          "Paternal Grandfather",
+          "Paternal Grandmother",
+          "Paternal Half-brother",
+          "Paternal Half-sister",
+          "Sister",
+          "Son",
+          "Unknown"
+        ]
+      },
+      "valueType": "string",
+      "displayName": "Relationship Type"
+    },
+    {
+      "description": "The self-reported gender of related individual.",
+      "name": "gender_of_relative",
+      "restrictions": {
+        "codeList": [
+          "Female",
+          "Male",
+          "Other",
+          "Unknown"
+        ]
+      },
+      "valueType": "string",
+      "displayName": "Gender of Relatives"
```

Comment:
Make "Relatives" singular since we're asking the gender of just one relative (ie. not an array field).